cdp4jsonserializer/src/test/java/cdp4jsonserializer/Cdp4JsonSerializerImplTest.java

```diff
@@ -255,4 +282,110 @@ boolean areEqualBySumOfBytes(String s1, String s2) {
     System.out.println(s1SumOfBytes + "--" + s2SumOfBytes);
     return s1SumOfBytes == s2SumOfBytes;
   }
+
+  private static String getJsonString() {
+    return String.join("\\n",
+        "{\"widget\": {",
+        "\"debug\": \"on\",",
+        "\"window\": {",
+        "\"title\": \"Sample Konfabulator Widget\",",
+        "\"name\": \"main_window\",",
+        "\"width\": 500,",
+        "\"height\": 500",
+        "},",
+        "\"image\": {",
+        "\"src\": \"Images/Sun.png\",",
+        "\"name\": \"sun1\",",
+        "\"hOffset\": 250,",
+        "\"vOffset\": 250,",
+        "\"alignment\": \"center\"",
+        "},",
+        "\"text\": {",
+        "\"data\": \"Click Here\",",
+        "\"size\": 36,",
+        "\"style\": \"bold\",",
+        "\"name\": \"text1\",",
+        "\"hOffset\": 250,",
+        "\"vOffset\": 100,",
+        "\"alignment\": \"center\",",
+        "\"onMouseUp\": \"sun1.opacity = (sun1.opacity / 100) * 90;\"",
+        "}",
+        "}",
+        "}");
+  }
+
+  private static String getXmlString() {
+
+    return String.join("\\n",
```

Comment:
According to this website (https://www.baeldung.com/java-string-newline), a newline character in java is also \n.
I think \\\n is something else: a "real" backslash followed by the letter 'n'.